src/create-config.ts

```diff
@@ -21,9 +21,15 @@ const questions = (choices: Choice[]): PromptObject[] => {
     },
     {
       type: 'confirm',
-      name: 'airnodeAuthorizer',
-      message: 'Do you want to include the Airnode Authorizer?',
-      initial: false,
+      name: 'gcp',
+      message: 'Do you want to create a configuration for GCP?',
```

Comment:
No need to clarify that this is only for Airnode config file, right?

Reply:
I guess I could mention this being only for airnode